docs/install-1.6.md

```diff
@@ -146,28 +146,37 @@ start over, check out the "Final Cleanup" section in the
 [walkthrough document](./walkthrough-1.6.md). Follow those instructions before 
 you start over.
 
-## Step 3 - Configuring `kubectl` to Talk to the API Server
+# Step 3 - Configuring `kubectl` to Talk to the API Server
 
 To configure `kubectl` to communicate with the service catalog API server, we'll have to
 get the IP address that points to the `Service` that sits in front of the API server pod(s).
 If you installed the catalog with one of the `helm install` commands above, then this service 
 will be called `catalog-catalog-apiserver`, and be in the `catalog` namespace. 
 
-### Notes on Getting the IP Address
+## Getting the IP Address of the API server
 
 How you get this IP address is highly dependent on your Kubernetes installation
-method. Regardless of how you do it, do not use the Cluster IP of the 
+method. Regardless of how you do it, do not use the `CLUSTER-IP` of the 
```

Comment:
Why ALL CAPS for this?

Reply:
`kubectl get svc` shows 'CLUSTER-IP'.  And, i used 'EXTERNAL-IP' below as well. Do you prefer 'Cluster IP' and 'External IP'?

Reply:
Ah, maybe we could insert a copy of `kubectl get svc` then. It does make sense in context.

Reply:
Updated. Please take another look. Thanks!